src/main.py

```
@@ -27,18 +27,14 @@
 from src.security import create_admin_jwt
 
 
-def get_log_level(env_var="LOG_LEVEL", default="INFO"):
+def get_log_level():
```

Ruff (0.11.9)

Missing return type annotation for public function `get_log_level`

(ANN201)